js/private/js_run_binary.bzl

```diff
@@ -30,6 +30,9 @@ def js_run_binary(
         silent_on_success = True,
         copy_srcs_to_bin = True,
         log_level = None,
+        mnemonic = "JsRunBinary",
```

Comment:
maybe sth like "UnknownJsBinary" to suggest that it's something a user ought to be setting

Reply:
hmmm. not sure on that one. maybe if we set a good mnemonic for all of the generated "bins" then that would make sense otherwise it would be everywhere.

Reply:
lets leave it as a follow-up